pcsx2/USB/usb-python2/usb-python2.cpp

```diff
@@ -570,9 +635,25 @@ namespace usb_python2
 					// LAMP_OUT also gets spammed so only send updates when something changes
 					if (curLightCabinet != s->f.oldLightCabinet)
 						mm_sendDDRMiniMaidUpdate();
+				}
 
-					s->f.oldLightCabinet = curLightCabinet;
+				if (s->isUsingBtoolLights)
+				{
+					//TODO: Set the player button lights using hdxs
+					curLightCabinet |= (((s->buf[5] & 0xf3) | 0x73) == 0x73) ? 1 << LIGHT_P1_UPPER_LAMP : 0;
+					curLightCabinet |= (((s->buf[5] & 0xf3) | 0xb3) == 0xb3) ? 1 << LIGHT_P1_LOWER_LAMP : 0;
+					curLightCabinet |= (((s->buf[5] & 0xf3) | 0xd3) == 0xd3) ? 1 << LIGHT_P2_UPPER_LAMP : 0;
+					curLightCabinet |= (((s->buf[5] & 0xf3) | 0xe3) == 0xe3) ? 1 << LIGHT_P2_LOWER_LAMP : 0;
+					curLightCabinet |= (((s->buf[5] & 0xf3) | 0xf2) == 0xf2) ? 1 << LIGHT_P1_MENU : 0;
+					curLightCabinet |= (((s->buf[5] & 0xf3) | 0xf1) == 0xf1) ? 1 << LIGHT_P2_MENU : 0;
+
+					if (curLightCabinet != s->f.oldLightCabinet && m_ddr_io_set_lights_p3io)
```

Comment:
Remove this `m_ddr_io_set_lights_p3io` check since it should be included in the `isUsingBtoolLights` variable after the previously suggested change.

```suggestion
					if (curLightCabinet != s->f.oldLightCabinet)
```